docs/docusaurus/docs/core/trigger_actions_based_on_results/_examples/create_a_custom_action.py

```diff
@@ -30,18 +30,27 @@ class MyCustomAction(ValidationAction):
     type: Literal["my_custom_action"] = "my_custom_action"
     # </snippet>
 
-    # 3. Override the `run()` method to perform the desired task.
+    # 3. Optionally add any additional fields your Action requires at runtime.
```

Comment:
Suggestion to follow the [style guide for presenting optional steps](https://greatexpectations.atlassian.net/wiki/spaces/DVRL/pages/657915907/Great+Expectations+style+guide)

```suggestion
    # 3. Optional. Add any additional fields your Action requires at runtime.
```